### ISSUE TITLE
__docgenInfo does not exist on FunctionComponent

### ISSUE THREAD

**Issue by jonathanloske** [atanasster/webpack-react-docgen-typescript#2]
FunctionComponents seem to not be handled correctly.

I have a component with the following code:

```tsx
import React, { FC } from "react";

const Font: FC<{className?: string}> = (props) => {
	return (
		<div>
			{props.children}
		</div>
	);
};

export { Font };
```

Using this results in `TS2339: Property '__docgenInfo' does not exist on type 'FunctionComponent<{ className?: string; }>'.`

I'm using a custom webpack config with a configuration as indicated in the README. Using Storybook 5.2.6

**Comment by atanasster** [comment]
Thanks @jonathanherdt - I added now a `ts-ignore` in the generated code, hopefully its fixed in  `0.9.5`

**Comment by jonathanloske** [comment]
It works, thank you! 😊
